press/article.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+In the recent react project we needed an environment to create, style and test our components without having a server-side backend code. We decided to use storybook that allows us to develop UI components outside your app and allow other people in your team to work on them. 
+
+The most important benefits of using storybook are :
+1. Completely isolate the environment for your components
+2. HMR — hot module replacement
+3. Support for CSS
+4. Multiple add-ons like accessibility, readme etc
+
+After using storybook in our project for a while,  we needed a way to display the CSS code samples that the component used inside the storybook panel, after looking on the internet we didn't find anything, so we decided to create our own add-on called storybook-addon-code. 
```

Comment:
'after looking on the internet' sounds to much like the 90s :) . Rephrase: 'after doing our fair share of research into existing solutions we've came up empty....'